src/Bookings.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
   const search = searchVal => {
```

Comment:
This is the best practise I think. Notice how all the code is wrapped in one function. We should always do this as it is much easier to read. When we start working with hundreds of components, using this method is very useful! So I would defiantly encourage you to start doing this now.